Sources/PrimerSDK/Classes/Core/Analytics/AnalyticsEvent.swift

```diff
@@ -271,6 +266,7 @@ extension Analytics.Event {
             case cancel                     = "CANCEL"
             case cardHolder                 = "CARD_HOLDER"
             case cardNumber                 = "CARD_NUMBER"
+            case cardNetwork                = "CARD_NETWORK"
```

Comment:
I would like to see this spaces removed, not a must, just the way I like to write code.

Reply:
Won't this be picked up by swiftlint?

Reply:
define `picked up`

Reply:
I would expect that swiftlint can detect + fix this kind of formatting if we don't want to support it.

Here I've just formatted it to be in keeping with the rest of the file, but if we don't want this formatting going forward we should use a swiftlint rule to enforce it. We can use that same rule to fix the issue using swiftlint.